index.js

```diff
@@ -28,29 +28,32 @@ const getEndpointSnippets = function (openApi, path, method, targets, values) {
     values = {};
   }
 
-  const har = OpenAPIToHar.getEndpoint(openApi, path, method, values);
-
-  const snippet = new HTTPSnippet(har);
+  const hars = OpenAPIToHar.getEndpoint(openApi, path, method, values);
 
   const snippets = [];
-  for (let j in targets) {
-    const target = formatTarget(targets[j]);
-    if (!target) throw new Error('Invalid target: ' + targets[j]);
-    snippets.push({
-      id: targets[j],
-      title: target.title,
-      content: snippet.convert(
-        target.language,
-        typeof target.library !== 'undefined' ? target.library : null
-      ),
-    });
+  for (const har of hars) {
+    const snippet = new HTTPSnippet(har);
+    for (let j in targets) {
+      const target = formatTarget(targets[j]);
+      if (!target) throw new Error('Invalid target: ' + targets[j]);
+      snippets.push({
+        id: targets[j],
+        mimeType: (har.comment ? har.comment : undefined),
+        title: target.title,
+        // add information to make unique
+        content: snippet.convert(
+          target.language,
+          typeof target.library !== 'undefined' ? target.library : null
+        ),
+      });
+    }
   }
 
   return {
-    method: har.method,
-    url: har.url,
-    description: har.description,
-    resource: getResourceName(har.url),
+    method: hars[0].method,
```

Comment:
Is method, url, and description always the same for all elements in hars array? Is this why you can use hars[0]?

Reply:
Yes those are identical for all elements in the array.